src/app/views/detail-idea/detail-idea.component.ts

```diff
@@ -49,10 +55,25 @@ export class DetailIdeaComponent {
   }
 
   onSubmitComment(){
-    // TODO: Save comment in DB
+    const comment: Comment = {
+      comment: this.commentContent.trim()
+    };
 
-    // Clear textarea content
-    this.commentContent = "";
+    this.loading = true;
+    this.commentsService.createComment(comment, this.idea.id!).subscribe({
+      next: () => {
+        this.commentContent = "";
+        this.getIdea();
```

Comment:
Instead of refresh all the page, obtain the comment from the response of the call and add it to the idea comments.